src/views/pxss_one.ecr

```diff
@@ -9,7 +9,7 @@
   </head>
   <body>
     <h1>Please leave a comment:</h1>
-    <form id="form" action="/pxss_one" method="post">
+    <form id="form" method="post">
       <label for="name">Name</label> <input type="text" id="name" name="name">
```

Comment:
We have same bug under `csrf` if you could also tend to that in this PR

Reply:
yas